Makefile

```diff
@@ -60,7 +60,7 @@ $(GOLANGGCI_LINT): install
 check-linter: ## Install and run golang linter
 check-linter: $(GOLANGGCI_LINT)
 	# Run golangci-lint linter
-	golangci-lint run -n
+	golangci-lint run
```

Comment:
Explicitly specify `-c .golangci.yml` here?

Reply:
Sure, that makes it obvious to the person reading this code that we're using that config file.